src/cmd/viper.go

```diff
@@ -221,4 +222,7 @@ func setDefaults() {
 
 	// Deploy opts that are non-zero values
 	v.SetDefault(VPkgDeployTimeout, config.ZarfDefaultTimeout)
+
+	// Package publish opts that are non-zero values
+	v.SetDefault(VPkgPublishRetries, 1)
```

Comment:
Any particular reason why not sticking with the `config.ZarfDefaultRetries` which is currently 3 and applies everywhere else?